### PR TITLE
feat: track user goals with google analytics

### PR DESCRIPTION
**Pull request by rihp**
It's still not registering it might take us merging it to main before seeing it live

### CODE REVIEW

**Comment by rihp** [comment]
we might have to add a disclaimer before adding this one

File: apps/browser/src/components/Chat/Chat.tsx [hunk 1]
```diff
@@ -93,14 +100,33 @@ const Chat: React.FC<ChatProps> = ({ evo, onMessage, messages, goalAchieved }: C
   }, [evoRunning, evoItr]);
 
   const handleSend = async () => {
+    // Your existing code
```
Comment (nerfZael) [inline apps/browser/src/components/Chat/Chat.tsx:103]: We can remove this :D

File: apps/browser/src/components/Chat/Chat.tsx [hunk 2]
```diff
@@ -93,14 +100,33 @@ const Chat: React.FC<ChatProps> = ({ evo, onMessage, messages, goalAchieved }: C
...
     onMessage({
       text: message,
       user: "user"
     });
     setSending(true);
     setMessage("");
     setEvoRunning(true);
+  
+    // Google Analytics Event Tracking
```
Comment (nerfZael) [inline apps/browser/src/components/Chat/Chat.tsx:112]: I'd move this to a separate function in a separate file

**Comment by nerfZael** [comment]
We should research whether or not we need a disclaimer for this

**Review by nerfZael — CHANGES_REQUESTED** [review]

**Comment by nerfZael** [comment]
Looks good, but I think we need the disclaimer :S

**Comment by rihp** [comment]
I've added a disclaimer at the bottom that has to be agreed to before inputting a goal
image

**Comment by rihp** [comment]
image

**Pull request closed without merging**